src/main/scala/beam/router/r5/FrequencyAdjustingNetworkCoordinator.scala

```diff
@@ -59,19 +69,35 @@ case class FrequencyAdjustingNetworkCoordinator(beamConfig: BeamConfig) extends
     }.head
   }
 
+  def getTripIdForRouteIdAtTime(routeId: String, startTime: Int, endTime: Int): String = {
+    feeds.map { feed =>
+      val allTrips = feed.trips.asScala
+      val routeTrips = allTrips.values.groupBy(_.route_id)(routeId)
+      val orderedFilteredStops = routeTrips
+        .flatMap { trip =>
+          feed.getOrderedStopTimesForTrip(trip.trip_id).asScala.toVector
+        }
+        .filter { stopTime =>
+          stopTime.arrival_time >= startTime
+        }
+        .toVector
+      orderedFilteredStops.head.trip_id
+    }.head
+  }
+
   def adjustTripFrequency(adjustmentInput: FrequencyAdjustmentInput): AddTrips.PatternTimetable = {
 
     val entry = new AddTrips.PatternTimetable
     entry.headwaySecs = adjustmentInput.headwaySecs
     entry.startTime = adjustmentInput.startTime
     entry.endTime = adjustmentInput.endTime
-    entry.sunday = true
-    entry.monday = entry.sunday
-    entry.tuesday = entry.monday
-    entry.wednesday = entry.tuesday
-    entry.thursday = entry.wednesday
-    entry.friday = entry.thursday
-    entry.saturday = entry.friday
+    entry.sunday = false
```

Comment:
Just wanted to point this out in case it was something left from testing. If it's indeed wanted to be hard coded then that's fine - I at least wanted to make sure it wasn't a mistake